src/main/java/offtop/AudioTransformation/Services/ProducerService.java

```diff
@@ -0,0 +1,30 @@
+package offtop.AudioTransformation.Services;
+
+import offtop.AudioTransformation.Models.AudioData;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class ProducerService {
+    private static final Logger logger = LoggerFactory.getLogger(ProducerService.class);
+    private static final String TOPIC = "IncomingAudioEvent";
+ 
```

Comment:
get rid of comments in this file, also the sendMessage() method isn't being used anywhere else, please delete